README.md

```diff
@@ -1,6 +1,8 @@
 skopeo [![Build Status](https://travis-ci.org/projectatomic/skopeo.svg?branch=master)](https://travis-ci.org/projectatomic/skopeo)
 =
 
+![skopeo logo](https://cdn.rawgit.com/runcom/skopeo-1/add-logo/logo/skopeo.svg)
+
 _Please be aware `skopeo` is still work in progress and it currently supports only registry API V2_
```

Comment:
I know it's not this change, but please add an 'a' before "work in progress"

Reply:
> I know it's not this change, but please add an 'a' before "work in progress"

can you send a PR to fix that instead?